integreat_compass/cms/views/interactions/report_list_view.py

```diff
@@ -0,0 +1,66 @@
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...models import OfferVersion, Report
+
+
+@method_decorator(permission_required("cms.add_vote"), name="dispatch")
+class ReportListView(TemplateView):
+    """
+    View for the report list
+    """
+
+    template_name = "interactions/report_list.html"
+
+    def get_context_data(self, **kwargs):
+        r"""
+        Context for the report list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+        return super().get_context_data(**kwargs) | {
+            "reports": [
+                report
+                for report in Report.objects.filter()
+                if report.offer_version
+                == report.offer_version.offer.public_version
+                == report.offer_version.offer.latest_version
```

Comment:
```suggestion
```
Sorry to reopen this, I think this line needs to go, my bad. IMO it's OK for the report to stay until a new version has been approved or the current one rejected.

Reply:
Ah okay :+1:

Reply:
Is applied now! Thank you for all your help!!